### PR TITLE
build: Update scripts to support uploading a new icon

### PR DESCRIPTION
**Pull request by taylorvnoj**
## Motivations

The [Technical Guide (in Confluence)](https://jobber.atlassian.net/wiki/spaces/DES/pages/1658585445/Technical+Guide+Adding+an+icon+to+Jobber+s+design+system) for adding an icon to Atlantis was outdated due to recent package changes.   

### Added

Installed `ts-node`: `v10.9.1`

### Changed

Scripts within the `package.json` were updated:

`bootstrap` command was created to link all sub packages, which will run after an install or the `convertIcoMoon` command, allowing the icons to be shared from `design` to `components`

## Testing

There is currently a new icon named `drag` in IcoMoon which has not officially been brought into Atlantis.  This means you can generate/download the fonts locally right away to test the new icon will be added.

`npm ci`
`npm run convertIcoMoon -- /pathname/jobbericons-v1.0/selection.json`
See that the icon is now listed in `iconMap.ts`
List the new icon in `Icon.stories.tsx` so make sure Storybook compiles it correctly

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,10 +16,12 @@
   "scripts": {
     "build": "npm run storybook:build",
     "build:hooksIndex": "node scripts/createIndexFile.js ./packages/hooks/src",
+    "bootstrap": "npm run bootstrap --workspaces --if-present",
     "ci:release": "lerna publish --yes",
     "clean": "rm -rf storybook-static",
     "clean:packages": "rm -rf node_modules **/dist",
     "convertIcoMoon": "scripts/convertIcoMoon.ts",
+    "postconvertIcoMoon": "npm run bootstrap",
```
Comment (JeffMarvin) [inline package.json:24]: Consider moving this down to keep this list alphabetical.

File: package.json [hunk 2]
```diff
@@ -16,10 +16,12 @@
   "scripts": {
     "build": "npm run storybook:build",
     "build:hooksIndex": "node scripts/createIndexFile.js ./packages/hooks/src",
+    "bootstrap": "npm run bootstrap --workspaces --if-present",
```
Comment (JeffMarvin) [inline package.json:19]: I have a naive question. What is "bootstrap" actually? We're running `npm run bootstrap` which makes me think it is a script, but then the `bootstrap` script also calls `npm run bootstrap`... ??? 
- signed recursively confused

**Review by JeffMarvin — CHANGES_REQUESTED** [review]
Great QA steps, thanks! 
It all works, makes sense. Great updates to the instructions on the wiki too! 
I had one question and one small suggestion to consider.

File: package.json (hunk 2 above)
Comment (JeffMarvin) [reply package.json:19]: Ah, I found it!
`bootstrap` is a script for each of the underlying "workspaces" which we are triggering here. Ok, I get it. 
(https://docs.npmjs.com/cli/v7/using-npm/workspaces#ignoring-missing-scripts)

Comment (taylorvnoj) [reply package.json:19]: You got it! I tried to briefly describe that in the PR description but didn't go too in depth.

**Review by JeffMarvin — APPROVED** [review]

**Pull request merged**